### DISCUSSION TITLE
Having issue with github actions.

### DISCUSSION THREAD

**Discussion by salmanfarisvp** [gethyas/hyas#465] [General]
Hi, I have a repo https://github.com/QTPYLabs/QTPYLabs.gitlab.io and enabled the GitHub action but it's not running and the GitHub page is serving from the README.md file. 

![image](https://github.com/gethyas/hyas/assets/11945360/2ed42a88-c4dd-4611-afc3-f2924a2a0735)

The page is working fine from local. Thanks in advance.

**Comment by h-enk** [comment]
You're using a CNAME file —  did you set [DNS records for your custom domain](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#dns-records-for-your-custom-domain)?

**Reply by h-enk** [reply]
Looks like it's working? http://qtpylabs.com/

**Reply by salmanfarisvp** [reply]
Hi h-enk. Thanks for the response. The issue was fixed. I put I wrongly put the folder name **/.github/workflow** instead of the **main/.github/workflows**
